### PR TITLE
[iOS] Fix issue updating Slider Maximum value

### LINKED ISSUE #5160: [iOS] Slider's MaximumValue does not change after initial assignment
### Description

Once the initial MaximumValue of Slider is set it does not change on iOS. 

### Steps to Reproduce

Run the repro project and wait for 5 seconds. Now the Slider will completely slide in one go because the MaximumValue  is still 1 even when the new Value of 100 is assigned.
[MauiBugSample.zip](https://github.com/dotnet/maui/files/8212452/MauiBugSample.zip)
 

### Version with bug

Preview 13 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Fix issue updating Slider Maximum value on iOS.

fix-5160

### Issues Fixed

Fixes #5160

### CODE REVIEW

**Review by rmarinho — DISMISSED** [review]
Tests are failing on Android

**Comment by jsuarezruiz** [comment]
> Tests are failing on Android

Updated. Running the build again now.

**Review by hartez — APPROVED** [review]

**Comment by rmarinho** [comment]
Tests are still failing @jsuarezruiz

**Review by mattleibow — DISMISSED** [review]

File: src/Core/tests/DeviceTests/Handlers/Slider/SliderHandlerTests.Android.cs [hunk 1]
```diff
@@ -42,8 +42,15 @@ public async Task ThumbImageSourceInitializesCorrectly()
...
 		}
 
-		double GetNativeMaximum(SliderHandler sliderHandler) =>
-			GetNativeSlider(sliderHandler).Max;
+		double GetNativeMaximum(SliderHandler sliderHandler)
+		{
+			if (OperatingSystem.IsAndroidVersionAtLeast(26))
+			{
+				return GetNativeSlider(sliderHandler).Max;
+			}
```
[the remark this answers is not in the document]
Comment (jsuarezruiz) [reply src/Core/tests/DeviceTests/Handlers/Slider/SliderHandlerTests.Android.cs:50]: Because it was introduced in API Level 26.

**Comment by rmarinho** [comment]
Tests still failing on Android

**Comment by jsuarezruiz** [comment]
> Tests still failing on Android

I will check the problem  but the PR introduce only iOS tests: src/Core/tests/DeviceTests/Handlers/Slider/SliderHandlerTests.iOS.cs

**Pull request merged**